.github/workflows/pypi_release.yml

```diff
@@ -0,0 +1,30 @@
+name: PypiRelease
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [3.8]
+
+    steps:
+    - uses: actions/checkout@master
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+
+    - name: Install dependencies
+      run: >-
+        python -m pip install --user --upgrade setuptools wheel
+    - name: Build
+      run: >-
+        python setup.py sdist bdist_wheel
+    - name: Publish distribution 📦 to PyPI
+      if: startsWith(github.event.ref, 'refs/tags')
```

Comment:
so it is triggered by adding tag but then the pushing version is taken from `setup.py` right?

Reply:
instead of this if you can just do 
```
on:
    tags: true
```

Reply:
I'm not sure if this works. I kept it like this, because this is, how they referenced it in their readme:
https://github.com/pypa/gh-action-pypi-publish

Reply:
so lets add the link to the config